anvil/src/eth/api.rs

```diff
@@ -71,6 +73,62 @@ use super::backend::mem::BlockRequest;
 /// The client version: `anvil/v{major}.{minor}.{patch}`
 pub const CLIENT_VERSION: &str = concat!("anvil/v", env!("CARGO_PKG_VERSION"));
 
+#[derive(Debug, Eq, PartialEq, serde::Serialize, serde::Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct NodeInfo {
+    current_block_number: U64,
+    current_block_timestamp: u64,
+    current_block_hash: H256,
+    hard_fork: SpecId,
+    transaction_order: String,
+    environment: NodeEnvironment,
+    fork_config: NodeForkConfig,
+}
```

Comment:
this is great, 
I'd like to move this to the core crate instead, I think there is a`core/types` mod.

Reply:
done in https://github.com/foundry-rs/foundry/pull/3779/commits/669c0dd6c9cc9725825824aef0f0c1be1bf65405